.github/workflows/build.yml

```diff
@@ -48,8 +48,7 @@ jobs:
       - run: npm run postinstall
       - run: npm run lint
       - run: npm run typecheck
-      - run: npm run test:unit
-      - run: npm run coverage
+      - run: env NODE_ENV=test npm run coverage
```

Comment:
`npm run coverage` starts test?

what's not obvious - 1. removing `test:unit` from CI
2. set NODE_ENV specifically ==='test'

Reply:
discussed "coverage" question

Reply:
test supposed to run with NODE_ENV = 'test' but new vitest pick value from environment (production) and react loads production bundle - without test utils like act() - which produces error